### ISSUE TITLE
Not banning invalid users?

### ISSUE THREAD

**Issue by lukeeexd** [fail2ban/fail2ban#2498]
### Environment:

_Fill out and check (`[x]`) the boxes which apply. If your Fail2Ban version is outdated, 
and you can't verify that the issue persists in the recent release, better seek support 
from the distribution you obtained Fail2Ban from_

- Fail2Ban version (including any possible distribution suffixes):
- OS, including release name/version: Ubuntu 18.04.2 LTS (GNU/Linux 4.15.0-52-generic x86_64)
- [x] Fail2Ban installed via OS/distribution mechanisms
- [x] You have not applied any additional foreign patches to the codebase
- [x] Some customizations were done to the configuration (provide details below is so)

### The issue:

Invalid user login attempts are not getting banned

#### Steps to reproduce

Try login as invalid user

#### Expected behavior

Get banned

#### Observed behavior

Not getting banned

#### Any customizations done to /etc/fail2ban/ configuration
jail.local
```[DEFAULT]
# Bantime of Perm
bantime = -1
# Max tries of 1
maxretry = 1

# Prevent abuse of SSH logins
[sshd]
enabled = true
logpath = /var/log/auth.log

```

#### Relevant parts of /var/log/fail2ban.log file:
_preferably obtained while running fail2ban with `loglevel = 4`_

```
2019-08-07 01:35:31,269 fail2ban.server         [31853]: INFO    Starting Fail2ban v0.10.2
2019-08-07 01:35:31,271 fail2ban.database       [31853]: INFO    Connected to fail2ban persistent database '/var/lib/fail2ban/fail2ban.sqlite3'
2019-08-07 01:35:31,271 fail2ban.jail           [31853]: INFO    Creating new jail 'sshd'
2019-08-07 01:35:31,276 fail2ban.jail           [31853]: INFO    Jail 'sshd' uses pyinotify {}
2019-08-07 01:35:31,278 fail2ban.jail           [31853]: INFO    Initiated 'pyinotify' backend
2019-08-07 01:35:31,278 fail2ban.filter         [31853]: INFO      maxLines: 1
2019-08-07 01:35:31,288 fail2ban.server         [31853]: INFO    Jail sshd is not a JournalFilter instance
2019-08-07 01:35:31,289 fail2ban.filter         [31853]: INFO    Added logfile: '/var/log/auth.log' (pos = 690509, hash = f7b3d86466e564c559800728faddb5316068cb43)
2019-08-07 01:35:31,290 fail2ban.filter         [31853]: INFO      encoding: UTF-8
2019-08-07 01:35:31,290 fail2ban.filter         [31853]: INFO      maxRetry: 1
2019-08-07 01:35:31,290 fail2ban.filter         [31853]: INFO      findtime: 600
2019-08-07 01:35:31,290 fail2ban.actions        [31853]: INFO      banTime: -1
2019-08-07 01:35:31,291 fail2ban.jail           [31853]: INFO    Jail 'sshd' started
```

#### Relevant lines from monitored log files in question:
auth.log
```
Aug  7 02:37:57 TWS sshd[399]: Invalid user developer from 116.196.119.177 port 36342
Aug  7 02:37:58 TWS sshd[399]: Received disconnect from 116.196.119.177 port 36342:11: Normal Shutdown, Thank you for playing [preauth]
Aug  7 02:37:58 TWS sshd[399]: Disconnected from invalid user developer 116.196.119.177 port 36342 [preauth]
Aug  7 02:38:16 TWS sshd[526]: Invalid user dial from 116.196.119.177 port 41329
```

**Comment by sebres** [comment]
Seems to be already fixed (#1581?), please update your filter... Or provide more info (e. g. fail2ban-version).

PoC with current 0.10:
```
$ fail2ban-regex 'Aug  7 02:37:57 TWS sshd[399]: Invalid user developer from 116.196.119.177 port 36342' sshd

Running tests
=============

Use   failregex filter file : sshd, basedir: /etc/fail2ban/
Use         maxlines : 1
Use      datepattern : Default Detectors
Use      single line : Aug  7 02:37:57 TWS sshd[399]: Invalid user develo...


Results
=======

Failregex: 1 total
|-  #) [# of hits] regular expression
|   6) [1] ^[iI](?:llegal|nvalid) user <F-USER>.*?</F-USER> from <HOST>(?: (?:port \d+|on \S+|\[preauth\])){0,3}\s*$
`-

...
Lines: 1 lines, 0 ignored, 1 matched, 0 missed
```

**Comment by lukeeexd** [comment]
I actually updated Ubuntu and it’s kernel and it started working straight away
